python_modules/dagster/dagster/_core/execution/context/op_execution_context.py

```diff
@@ -599,6 +599,14 @@ def is_subset(self):
     @public
     @property
     def selected_asset_check_keys(self) -> AbstractSet[AssetCheckKey]:
+        """Returns the set of asset check keys associated with the selected assets.
+
+        This property retrieves the asset check keys from the associated assets definition
+        if it exists. If there is no associated assets definition, it returns an empty set.
+
+        Returns:
+            AbstractSet[AssetCheckKey]: A set of asset check keys for the selected assets.
```

Comment:
I think AbstractSet is the type that satisfies mypy, but we are actually returning a Set? Maybe someone like @sryza knows better here though

Reply:
Removed `Returns` section to be aligned with other docstrings.